go/teams/chain.go

```diff
@@ -1518,18 +1530,50 @@ func (t *TeamSigChainPlayer) addInnerLink(
 							if err != nil {
 								return err
 							}
-							if _, err := prevState.GetLatestUVWithUID(uv.Uid); err != nil {
-								return err
+							addedUIDs[uv.Uid] = true
+							_, err = prevState.GetLatestUVWithUID(uv.Uid)
+							if err == nil {
+								// Found crypto member in previous
+								// state, we are good!
+								return nil
```

Comment:
This returns from `checkImpteamInvites`. Is that what's intended? Or is this meant to continue the loop?

Reply:
oh you are right, it should continue, not return right away! thanks